Chemistry/tests/SamplesTests/DocsInvokingChemistryLibrary.cs

```
@@ -0,0 +1,153 @@
+            // expressed in terms of Fermionic operators, to a qubit Hamiltonian,
+            // expressed in terms of Pauli matrices. This is an essential step
+            // for simulating our constructed Hamiltonians on a qubit quantum
+            // computer.
+            var jordanWignerEncoding = fermionHamiltonian.ToPauliHamiltonian(Pauli.QubitEncoding.JordanWigner);
+
+            // We also need to create an input quantum state to this Hamiltonian.
+            // Let us use the Hartree-Fock state.
+            var fermionWavefunction = fermionHamiltonian.CreateHartreeFockState(nElectrons);
+
+            // This Jordan-Wigner data structure also contains a representation 
+            // of the Hamiltonian and wavefunction made for consumption by the Q# algorithms.
+            var qSharpHamiltonianData = jordanWignerEncoding.ToQSharpFormat();
+            var qSharpWavefunctionData = fermionWavefunction.ToQSharpFormat();
+            var qSharpData = QSharpFormat.Convert.ToQSharpFormat(qSharpHamiltonianData, qSharpWavefunctionData);
+
+            Assert.True(fermionWavefunction.MCFData.Excitations.Keys.Single()
+                .ToIndices().SequenceEqual(new[] { 0, 1 })
+                );
+        }
+
+        [Fact]
+        static void LoadFromBroombridgeFile()
+        {
+            // This is the name of the file we want to load
+            var filename = @"LiH_0.2.yaml";
+            // This is the directory containing the file
+            var root = @"Molecules";
+
+            // This deserializes Broombridge.
+            var broombridge = Broombridge.Deserializers.DeserializeBroombridge($@"{root}\{filename}");
+
+            // Note that the deserializer returns a list of `ProblemDescriptions` instances 
+            // as the file might describe multiple Hamiltonians. In this example, there is 
+            // only one Hamiltonian. So we use `.First()`, which selects the first element of the list.
```

Wouldn't that be a good usecase for `.Single`? In the case that a file with multiple problem descriptions is loaded, that would cause an explicit exception to be raised instead of silently dropping data.